### DISCUSSION TITLE
How to close a document opened in split view

### DISCUSSION THREAD

**Discussion by vuggy17** [toeverything/AFFiNE#7564] [Q&A] [answered]
.

**Comment by IveTian** [comment, marked as answer]
Hi there!
I apologize for the bad experience with split view.
I understand your claim. All you have to do is tap on the blue drag hand and select “Close” in the expand menu.
I hope this helps!
![CleanShot 2024-07-22 at 14 57 35@2x](https://github.com/user-attachments/assets/ec6e8984-f667-4083-9e39-1b8ef5b35174)

**Reply by vuggy17** [reply]
Thank you for the quick reply. 

I initially thought it was just a drag indicator to move the window around, but I never realized I needed to tap it! 😂

**Reply by hwangdev97** [reply]
😂 We will improve this feature. Do you have any other suggestions?

**Reply by vuggy17** [reply]
I prefer doing something like Edge split screen feature do

Regarding the options, you could consider making them appear when users hover over the page header. 

![image](https://github.com/user-attachments/assets/8c2a44cb-3a9a-4371-93b8-ff69fc232a35)

**Reply by IveTian** [reply]
Thx for your advise :) that's why the split view is still in "Beta" stage. We will continue to improve.
